pkg/codegen/go/doc.go

```diff
@@ -88,8 +88,8 @@ func (d DocLanguageHelper) GetLanguageTypeString(pkg *schema.Package, moduleName
 		glog.Errorf("cannot calculate type string for type %q. could not find a package for module %q", t.String(), moduleName)
 		os.Exit(1)
 	}
-	if _, ok := t.(*schema.EnumType); ok {
-		return modPkg.inputType(t)
+	if typ, ok := t.(*schema.EnumType); ok {
+		return modPkg.tokenToEnum(typ.Token)
```

Comment:
What happens if we delete these lines entirely? I have a feeling that the call to `typeString` below will do the right thing.

Reply:
Indeed!